src/Assets/AugmentedAssetContainer.php

```diff
@@ -17,6 +17,7 @@ public function keys()
             'search_index',
             'api_url',
             'assets',
+            'max_size',
```

Comment:
@jasonvarga I don't understand why this is needed, but without it the tests don't pass. With it, unrelated tests fail.